### ISSUE TITLE
jwt security doesn't behave properly in swagger

### ISSUE THREAD

**Issue by matteosuppo** [goadesign/goa#663]
By generating the swagger documentation for https://github.com/goadesign/examples/blob/master/security/design/jwt.go

you obtain

```yaml
consumes:
- application/json
- application/x-www-form-urlencoded
definitions:
  Success:
    description: The common media type to all request responses for this example (default
      view)
    example:
      ok: true
    properties:
      ok:
        description: Always true
        example: true
        type: boolean
    required:
    - ok
    title: 'Mediatype identifier: application/vnd.goa.examples.security.success'
    type: object
host: localhost:8080
info:
  description: 'This artifical API shows 4 different ways to secure API endpoints:
    using basic auth, shared secret header, JWT or OAuth2'
  version: ""
paths:
  /jwt:
    get:
      description: |-
        This action is secured with the jwt scheme

        ** Required security scopes**:
          * `api:access`
      operationId: jwt#secure
      parameters:
      - description: Force auth failure via JWT validation middleware
        in: query
        name: fail
        required: false
        type: boolean
      responses:
        "200":
          description: OK
          schema:
            $ref: '#/definitions/Success'
        "401":
          description: Unauthorized
      schemes:
      - http
      summary: secure jwt
      tags:
      - jwt
  /jwt/signin:
    post:
      description: Creates a valid JWT
      operationId: jwt#signin
      responses:
        "204":
          description: No Content
          headers:
            Authorization:
              description: Generated JWT
              type: string
        "401":
          description: Unauthorized
      schemes:
      - http
      security:
      - SigninBasicAuth: []
      summary: signin jwt
      tags:
      - jwt
  /jwt/unsecure:
    get:
      description: This action does not require auth
      operationId: jwt#unsecure
      responses:
        "200":
          description: OK
          schema:
            $ref: '#/definitions/Success'
      schemes:
      - http
      summary: unsecure jwt
      tags:
      - jwt
produces:
- application/json
- application/x-www-form-urlencoded
responses:
  OK:
    description: OK
    schema:
      $ref: '#/definitions/Success'
  Unauthorized:
    description: Unauthorized
schemes:
- http
securityDefinitions:
  SigninBasicAuth:
    type: basic
  jwt:
    description: |2-


      **Security Scopes**:
        * `api:access`: API access
    in: header
    name: Authorization
    type: apiKey
swagger: "2.0"
```

The /jwt path is missing something like

```yaml
security:
  - jwt: []
```

**Comment by raphael** [comment]
Indeed, thanks for the report!

**Comment by raphael** [comment]
Fixed in evil PR #666
